AstroCal/view/console_output.py

```diff
@@ -3,41 +3,49 @@
 # Imports
 from datetime import datetime, timedelta
 from gettext import find
-from control import control
+from pickle import TRUE
+from AstroCal.control import control
 import sys
+import os
 
 
 # Main Menu
 def main_menu():
-    print('✧ ･ﾟ * ✧  ASTRO CALANDER  ✧ ･ﾟ * ✧ ･ﾟ \n')
```

Comment:
I think we should revert and trademark CALANDER

Reply:
Done